neuror/main.py

```diff
@@ -284,7 +287,10 @@ def __init__(self,
                                 for section in self.neuron.iter())
 
         self.info = dict()
-        apical_section_id, _ = apical_point_section_segment(self.neuron)
+        if apical_point:
+            apical_section_id = point_to_section_segment(self.neuron, apical_point)[0] - 1
```

Comment:
Compared to https://github.com/BlueBrain/NeuroR/pull/53 , this `-1` was moved here. Does it mean that the function `morph_tool.spatial.point_to_section_segment` is not correct?

Reply:
I do the same as this: https://github.com/BlueBrain/morph-tool/blob/master/morph_tool/apical_point.py#L33, right? I don't fully understand this index inconsistency tbh. @wizmer may be able to clarify it.

Reply:
Because we want to return the MorphIO ID
You can add this comment in the code
`MorphIO ID = NeuroM ID - 1`